Add note about ports range format (#2179)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012r2-ps/hyper-v/Add-VMNetworkAdapterExtendedAcl.md

```

### -LocalPort

> Specifies the local port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).

Reviewer: ```suggestion
Specifies the local port for the ACL. A port range format can be used also (i.e. "49152-49182", for example).
```

> Specifies the local port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).

Reviewer 2: ```suggestion
Specifies the local port for the ACL. A port range format can also be used (i.e. "49152-49182", for example).
```

```

### -RemotePort

> Specifies the remote port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).

Reviewer: ```suggestion
Specifies the remote port for the ACL. A port range format can be used also  (i.e. "49152-49182", for example).
```

> Specifies the remote port for the ACL. Also a ports range format can be used (i.e. "49152-49182", for example).

Reviewer 2: ```suggestion
Specifies the local port for the ACL. A port range format can also be used (i.e. "49152-49182", for example).
```
